dracut/fetch-kickstart-net.sh

```diff
@@ -87,7 +88,13 @@ for kickstart in \$locations; do
         break
     else
         warn_critical "anaconda: failed to fetch kickstart from \$kickstart"
+	errors=1
     fi
 done
+
+if [[ \$errors -eq 1 ]]; then
```

Comment:
This condition will add a delay also for cases with multiple `inst.ks=`, where the first kickstart cannot be fetched, but the second one is successfully downloaded.

Reply:
Oh you can do that? Oh my, is there a way for me to solve this issue then?

Reply:
If `fetch_url` in the `for` loop above succeeds, then ensure no delay is added.

Reply:
Oh right I mis-read your statement, sure, good catch. Rebased.